src/scales/scale.time.js

```diff
@@ -758,7 +758,7 @@ module.exports = Scale.extend({
 		var exampleLabel = me.tickFormatFunction(exampleTime, 0, [], format);
 		var tickLabelWidth = me.getLabelWidth(exampleLabel);
 		var innerWidth = me.isHorizontal() ? me.width : me.height;
-		var capacity = Math.floor(innerWidth / tickLabelWidth);
+		var capacity = Math.floor(innerWidth / tickLabelWidth) - 1;
```

Comment:
I don't think it's correct because you should still be able to fit one more label:

```
innerWidth >= Math.floor(innerWidth / tickLabelWidth) * tickLabelWidth
```

The issue may be related to how we compute `getLabelWidth()` which may be different from the one [used by the auto skip](/chartjs/Chart.js/blob/master/src/core/core.scale.js#L427) feature.

Reply:
You are right, this is not the issue. The issue is we are not considering padding in `innerWidth`. Also ticks are build before `fit`, so padding is actually not calculated yet. -1 gives similar result as the padding consideration would, but its not strictly correct.